src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -114,4 +115,11 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+  /** Returns the total number of messages for all users. */
+  public int getTotalMessageCount() {
+    Query query = new Query("Message");
+    PreparedQuery results = datastore.prepare(query);
+    return results.countEntities(FetchOptions.Builder.withLimit(1000));
```

Comment:
Let's make the 1000 a static const int up at the top. We don't want magic numbers here.

Here's the reference:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names